docs/book/v4/migration/v3-to-v4.md

```diff
@@ -1,5 +1,47 @@
 # Migration from Version 3 to 4
 
-INFO: **Work in progress**
-We are currently working on the migration guide.
-We will update this page as soon as possible!
```

Comment:
Since this is a partial migration guide I suggest we keep this disclaimer until full migration guide is available